ipaplatform/suse/tasks.py

```diff
@@ -42,5 +43,158 @@ def set_nisdomain(self, nisdomain):
     def set_selinux_booleans(self, required_settings, backup_func=None):
         return False  # FIXME: Implement after libexec move
 
+    def configure_nsswitch_database(self, fstore, database, services,
+                                    preserve=True, append=True,
+                                    default_value=()):
+        """
+        Edits the specified nsswitch.conf database (e.g. passwd, group,
+        sudoers) to use the specified service(s).
+
+        Arguments:
+            fstore - FileStore to backup the nsswitch.conf
+            database - database configuration that should be ammended,
+                       e.g. 'sudoers'
+            service - list of services that should be added, e.g. ['sss']
+            preserve - if True, the already configured services will be
+                       preserved
+
+        The next arguments modify the behaviour if preserve=True:
+            append - if True, the services will be appended, if False,
+                     prepended
+            default_value - list of services that are considered as default (if
+                            the database is not mentioned in nsswitch.conf),
+                            e.g. ['files']
+        """
+
+        from ipapython.ipachangeconf import IPAChangeConf
```

Comment:
Why is this a local import instead of a module level import?

Reply:
This session has been removed by using the base module.